app/assets/stylesheets/revised/pages/landing/_signup.scss

```diff
@@ -41,47 +64,58 @@ $landing-page-targets: "shop", "school", "advocacy", "law_enforcement",
     .svgicon-body {
       fill: #fff;
     }
+
   }
 }
 
 .signup-landing-page {
+  padding-bottom: 40px;
+
   .trusted-by {
-    ul {
-      @include list-unstyled;
+    padding: 10px;
 
-      display: flex;
-      flex-wrap: wrap;
+    .header {
+      background-color: transparent;
+      text-align: center;
+      font-weight: 600;
+      font-size: 14px;
+      margin: 0;
+      padding: 0;
     }
 
-    li {
-      flex: 0 0 20%;
-
-      &.partner-logo-rectangle {
-        flex: 0 0 40%;
-      }
-
-      padding: $vertical-height (0.5 * $vertical-height) (2 * $vertical-height);
+    img {
+      display: block;
+      width: 100%;
+      height: auto;
+      padding-right: 15px;
+    }
+  }
+}
 
-      img {
-        display: block;
-        width: 100%;
-        height: auto;
-      }
+.landing-page {
+  @include media-breakpoint-down(lg) {
+    .banner-header {
+      padding-top: 100px !important;
```

Comment:
Why do you need the `!important`s here?

Reply:
Good nudge thanks -- I couldn't find where in our stylesheets the overridden rules are defined, but I refactored to work around that without the `!important`s